src/lib/connectors/redisConnector.ts

```diff
@@ -1,5 +1,6 @@
 import { type RedisClientType, createClient } from "redis";
-import { REDIS_URL } from "@src/config";
+import { REDIS_URL } from "@src/config.js";
+import { logMessage } from "../logger.js";
```

Comment:
For consistency, could we use the prefixed imports like so?
```suggestion
import { logMessage } from "@lib/logger.js";
```

Also very happy to see a proper logger get added.  I realized yesterday that all my `console.debug` statements were going to output to `stdout` with no way to control via a log level.

Reply:
yeah, autocomplete likes to use relative folder structure.
I can refactor